src/app/App.js

```diff
@@ -1,9 +1,24 @@
 import { render } from '../shared/dom';
+import ApiProvider from '../providers/ApiProvider';
 
-const template = `
-    <div>Hello world!</div>
+const template = (movies) => `
+    <div id="app">
+        <div>Hello world!</div>
+        <ol>
+            ${movies.reduce((r, movie) => r + `<li>${movie}</li>`, '')}
+        </ol>
+    </div>
 `;
 
-export const App = ({ renderOn, options }) => {
-  render({ on: renderOn, html: template });
+export const App = async ({ renderOn }) => {
+    let movies = [];
+    const { results } = await ApiProvider.search('Fast & Furious');
+    console.debug(data);
```

Comment:
Do we need this now?

Reply:
It is just an example, but ok imma remove it